blazor/datagrid/sorting.md

```diff
@@ -273,6 +273,86 @@ The following GIF represents custom SortComparer for CustomerID column. When the
 N> The SortComparer property will work only for local data.
 <br/> When using the column template to display data in a column, you will need to use the [Field](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_Field) property of [GridColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html) to work with the [SortComparer](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_SortComparer) property.
 
+## AllowUnSort property
+
+To set the [AllowUnsort](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridSortSettings.html#Syncfusion_Blazor_Grids_GridSortSettings_AllowUnsort) to false in [GridSortSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_SortSettings), the action of clicking on a column header to revert the grid to its original unsorted layout will be restricted.
```

Comment:
does this content have any meaning. please understand the usage of document. Proper title, purpose or usage of the documentation. explanation of example below. then code example. sample